tests/unit/pages/quiz/Quiz.spec.jsx

```diff
@@ -94,6 +95,8 @@ describe('QuizPage for student', () => {
       fireEvent.click(confirmButton)
```

Comment:
```suggestion
      fireEvent.click(confirmButton)
```

Why did you use **act** for this button and others? As I know, we don't use it. It's enough to do 

```
fireEvent.click(confirmButton)
``` 
without act